packages/wrangler/src/__tests__/versions/versions.list.test.ts

```diff
@@ -41,36 +41,36 @@ describe("versions list", () => {
 			await expect(result).resolves.toBeUndefined();
 
 			expect(std.out).toMatchInlineSnapshot(`
-			"Version ID:  40000000-0000-0000-0000-000000000000
-			Created:     2021-01-01T00:00:00.000Z
-			Author:      Jean-Luc-Picard@federation.org
-			Source:      Upload
-			Tag:         -
-			Message:     -
+				"Version ID:  40000000-0000-0000-0000-000000000000
+				Created:     2021-01-01T00:00:00.000Z
+				Author:      Jean-Luc-Picard@federation.org
+				Source:      Upload
+				Tag:         -
+				Message:     -
 
-			Version ID:  30000000-0000-0000-0000-000000000000
-			Created:     2021-02-02T00:00:00.000Z
-			Author:      Kathryn-Janeway@federation.org
-			Source:      Rollback
-			Tag:         -
-			Message:     Rolled back for this version
+				Version ID:  10000000-0000-0000-0000-000000000000
+				Created:     2021-01-04T00:00:00.000Z
+				Author:      Jean-Luc-Picard@federation.org
+				Source:      Rollback
+				Tag:         -
+				Message:     -
 
-			Version ID:  20000000-0000-0000-0000-000000000000
-			Created:     2021-02-03T00:00:00.000Z
-			Author:      Kathryn-Janeway@federation.org
-			Source:      Wrangler 🤠
-			Tag:         -
-			Message:     -
+				Version ID:  30000000-0000-0000-0000-000000000000
+				Created:     2021-02-02T00:00:00.000Z
+				Author:      Kathryn-Janeway@federation.org
+				Source:      Rollback
+				Tag:         -
+				Message:     Rolled back for this version
 
-			Version ID:  10000000-0000-0000-0000-000000000000
-			Created:     2021-01-04T00:00:00.000Z
-			Author:      Jean-Luc-Picard@federation.org
-			Source:      Rollback
-			Tag:         -
-			Message:     -
+				Version ID:  20000000-0000-0000-0000-000000000000
+				Created:     2021-02-03T00:00:00.000Z
```

Comment:
These snapshots look out of order. Even considering the reversal

Reply:
The fake UUIDs are just done based on how the old order used to be, if you looked at the created it's actually all correct.

I'll update the fake UUIDs so it's less confusing

Reply:
Argh I was only looking at the days not the months 🤦